crates/uv-dispatch/src/lib.rs

```diff
@@ -185,8 +187,9 @@ impl<'a> BuildContext for BuildDispatch<'a> {
             let site_packages = SitePackages::from_executable(venv)?;
 
             let Plan {
-                local,
+                cached,
                 remote,
+                local: _,
```

Comment:
I currently don't do any reporting on these yet, which matches our already-satisfied installation behavior in general.

I'd like to explore adding output in a follow-up.

Reply:
In theory, these should always be empty...

Reply:
Do you mean just in dispatch or..?

I populate `local` at https://github.com/astral-sh/uv/blob/5a363fc2d4bccd2e049acc1fcfa18ed24fbd6e4a/crates/uv-installer/src/plan.rs#L177

Reply:
I just mean in `dispatch`, since it's a fresh virtual environment IIUC.

Reply:
Ah okay yeah that makes sense. I commented on an arbitrary line here.